### PR TITLE
Add button color variables

### PR DESCRIPTION
**Pull request by marcoambrosini**

### CODE REVIEW

**Comment by rullzer** [comment]
So... what is the status here?

**Comment by ma12-co** [comment]
> So... what is the status here?

Back on it these days :)

File: core/css/variables.scss [hunk 1]
```diff
@@ -81,6 +83,12 @@ $color-box-shadow: transparentize(nc-darken($color-main-background, 70%), 0.5) !
 $color-border: nc-darken($color-main-background, 7%) !default;
 // darker border like inputs or very visible elements
 $color-border-dark: nc-darken($color-main-background, 14%) !default;
+
+// Ripple effect
+$color-ripple: nc-darken($color-main-background, 30%);
+$color-ripple-primary: nc-lighten($color-primary-active, 20%);
```
Comment (juliushaertl) [inline core/css/variables.scss:89]: There is no such variable `$color-primary-active`

**Review by juliusknorr — APPROVED** [review]

File: core/css/inputs.scss [hunk 2]
```diff
@@ -24,7 +24,7 @@ $default-height: 34px;
 
 /* Simple selector to allow easy overriding */
 select,
-button,
+button:not(.button-vue),
```
Comment (artonge) [inline core/css/inputs.scss:27]: Is this to prevent the vue button from being styled from here ? Is this really necessary as it is probably scoped ?

Comment (artonge) [reply core/css/inputs.scss:27]: Hmm, ok, I thought that vue components could be isolated from external CSS. Maybe I am wrong :)

**Review by artonge — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by raimund-schluessler** [comment]
Can we backport it to stable21 and stable20 as well? Would help using the button component.

I will give backporting a try, let's see if it works.

**Pull request merged**